hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveMetastore.java

```diff
@@ -106,7 +115,9 @@ public void start(int poolSize) {
 
       TServerSocket socket = new TServerSocket(0);
       int port = socket.getServerSocket().getLocalPort();
-      this.hiveConf = newHiveConf(port);
+      initConf(conf, port);
```

Comment:
Nit: if `initConf` returned the configuration, this could be one line still: `this.hiveConf = initConf(conf, port);`

Reply:
Before the change the configuration was created in `newHiveConf()`. The new method changes values in the conf parameter. The change meant to emphasize that we are not creating a new conf, only changing values in the one got in the parameter list.

Shall we move back to create a new config and initialize that?

Reply:
I thought it was a necessary change to pass in the conf for some reason. I'm okay either way. I just wanted to point out a way to make fewer changes.

Reply:
If conf is passed in, I think the change to `initConf` makes sense. I just think the patch could be slightly smaller. Not a blocker, though.